src/content-handlers/iiif/modules/uv-openseadragoncenterpanel-module/OpenSeadragonCenterPanel.ts

```diff
@@ -75,6 +75,10 @@ export class OpenSeadragonCenterPanel extends CenterPanel<
     this.$viewer = $('<div id="' + this.viewerId + '" class="viewer"></div>');
     this.$content.prepend(this.$viewer);
 
+    //render navigator in a separate div to have more control over size and position
```

Comment:
Adding space for style consistency:

```suggestion
    // render navigator in a separate div to have more control over size and position
```